### DISCUSSION TITLE
DataGrid with nullable DateTime

### DISCUSSION THREAD

**Discussion by Marctic** [blazority/support#40] [Q&A] [answered]
I have a nullable DateTime field that would be better displayed as either blank when null or in a short date format when not. Is this something Blazority can handle today?

**Comment by karan-kang** [comment, marked as answer]
Yes, the Datagrid supports Nullable type. You have full control of the way you want to display data using the DatagridColumn fragment.

See the usage for this example, and take note of the date field: https://blazority.com/datagrid/structure
